components/molecules/Navigation.js

```diff
@@ -1,17 +1,42 @@
 import React from "react";
 import Link from "next/link";
+import styled from "styled-components";
+
+const Logo = styled("ul")`
```

Comment:
I'm not sure this name is quite right, the component contains all links - not just the logo.